README.md

```diff
@@ -11,6 +11,102 @@ This software is under active development and likely contains many bugs.
 Please open bugs on Github if you discover any issues with the software
 or documentation.
 
+Installation and Usage
+----------------------
+
+OnionGateway requires a working Go build environment. Once you have that
+configured you can build the `entry_proxy` and it's dependencies with the
+`go get` command. The binary will be built inside your `$GOPATH/bin`
+directory. You may need to add this directory to your shell `$PATH`
+environment variable.
+
+```bash
+go get github.com/DonnchaC/oniongateway
+sudo setcap 'cap_net_bind_service=+ep' $(which entry_proxy)
+entry_proxy
+```
+
+To improve performance, the server running the `entry_proxy` should have a Tor
+daemon which is running in `Tor2Web` mode. There are instructions for
+compiling Tor in this mode at https://github.com/globaleaks/Tor2web/wiki/Installation-Guide#build-tor-with-tor2web-mode-and-some-patches.
```

Comment:
IMHO, long reference can be moved to reference list (see below, near `[travis-page]`).